docs/docs/FAQ/Machine-Learning-FAQ.md

```diff
@@ -0,0 +1,125 @@
+
+See the [full discussion](https://github.com/immich-app/immich/discussions/5028) in GitHub for more information.
+
+</details>
+
+* For the smart search feature the best Clip model will be `ViT-H-14-378-quickgelu` or `vit-H-14-Qickgelu`.
+
+<details>
+  <summary>why is it?</summary>
+
+You might want to consider `ViT-H-14-quickgelu` over `ViT-H-14-378-quickgeludfn5b`
+because it takes much less processing power and the difference between the
+Average perf. on 38 datasets is really "nothing"
+to be precise a difference of 0.0118.
+And the advantage of ViT-H-14-quickgelu is that it requires much less processing power
+381.68 compared to 1054.05.
+
+Sources:
+[Discord](https://discord.com/channels/979116623879368755/1174254430221254738),
+[openclip results](https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_results.csv#L3),
+[openclip retrieval results](https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_retrieval_results.csv).
+
+</details>
+
+<details>
+  <summary>How to apply?</summary>
+
+1. In the browser, log in to your Admin user
+2. Go to the url [http://localhost:2283/admin/system-settings?open=job-settings](http://localhost:2283/admin/system-settings?open=job-settings)
+3. Go to Machine Learning Settings
+4. change the models and save
+5. rerun the jobs of ML to take effect
+
+</details>
+
+:::note
+Changing a model does not delete the previous model, pay attention to delete the old model to save space in drive.
+Models are stored in IMMICH_MODEL-CACHE.
+:::
+
+### I would like to use the Clip model i created, how can i do that?
+
+you can't, because we have a [whitelist](https://github.com/immich-app/immich/blob/main/machine-learning/app/models/constants.py) of models in the server code that has each model’s [dimension size](https://github.com/immich-app/immich/blob/main/server/src/domain/smart-info/smart-info.constant.ts).  We use this to update the index to the right dim size when the model changes so custom models aren’t supported.
+The [export script](https://github.com/immich-app/immich/tree/main/machine-learning/export) in Immich Github page is meant for us to be able to automate exporting and uploading to Huggingface you can send a [pull request](https://github.com/immich-app/immich/pulls) via Github to add this model to Immich check if Immich already have this model in the list of models in [Huggingface](https://huggingface.co/immich-app) before you send pull request.
+([Sources](https://discord.com/channels/979116623879368755/1071165397228855327/1187088114599071855))
+
+### I don't speak English, is there a way I can use the Smart Serach in my language?
+
+Yes, you can get such an option, to do this you need to check what is the best model according to your language, you can do it by checking the [table](https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_multilingual_retrieval_results.csv).
```

Comment:
I thought of listing the name of a model with a high percentage of accuracy in Multilingual but I could not find the models
LABSE-Vit-L-14
and
XLM-Roberta-Large-Vit-L-14

In the open_clip performance report. That's why I left a link to the table so that people can check for themselves what would be the best option.
In any case if you think it is less relevant I will remove it.

Reply:
Those models are from M-CLIP, not OpenCLIP. I think this is a useful question, but it should be about what Immich supports. 

Also, the question is kinda funny. If they don't know English, they can't read the answer either 😄 

You can change it to "I want to be able to search in other languages besides English. How can I do that?" with the answer:

"This is supported. You can change to a multilingual model listed [here](https://huggingface.co/collections/immich-app/multilingual-clip-654eb08c2382f591eeb8c2a7) by going to Administration > Machine Learning Settings > Smart Search and replacing the name of the model. You can then search in over 100 languages."